assets/src/scss/styleguide.scss

```diff
@@ -177,17 +177,17 @@ code {
 // p.typeSpecimen::after  { content: "Font stack: #{ko-font(base)}"; }
 
 .sg-typeSpecimen--body {
-	font-size: ko-rem(30);
+	font-size: 30px;
```

Comment:
Ah ok – that makes sense if they don’t need to hit the modular scale functions ⚡️